app/src/features/surveys/observations/sampling-sites/SamplingSiteList.tsx

```diff
@@ -113,6 +114,68 @@ const SamplingSiteList = () => {
       }
     });
   };
+
+  const handleCheckboxChange = (sampleSiteId: number) => {
+    setChecked((prev) => {
+      if (prev.includes(sampleSiteId)) {
+        return prev.filter((item) => item !== sampleSiteId);
+      } else {
+        return [...prev, sampleSiteId];
+      }
+    });
+  };
+
+  const handleBulkDeleteSampleSites = async () => {
+    await biohubApi.samplingSite
+      .deleteSampleSites(surveyContext.projectId, surveyContext.surveyId, checked)
+      .then(() => {
+        dialogContext.setYesNoDialog({ open: false });
+        setChecked([]);
+        surveyContext.sampleSiteDataLoader.refresh(surveyContext.projectId, surveyContext.surveyId);
+      })
+      .catch((error: any) => {
+        dialogContext.setYesNoDialog({ open: false });
+        setChecked([]);
+        dialogContext.setSnackbar({
+          snackbarMessage: (
+            <>
+              <Typography variant="body2" component="div">
+                <strong>Error Deleting Sampling Sites</strong>
+              </Typography>
+              <Typography variant="body2" component="div">
+                {String(error)}
+              </Typography>
+            </>
+          ),
+          open: true
+        });
+      });
+  };
+
+  const handleBulkDelete = () => {
```

Comment:
Nit: could rename this to `handlePromptConfirmBulkDelete` since it's named pretty similarly to the other callback